src/app/old/utils/commonFunctions.tsx

```diff
@@ -1,8 +1,11 @@
 const apiServer = `${window.location.protocol}//${window.location.hostname}${window.API_PORT ? `:${window.API_PORT}` : ''}`;
 
+const flightCtlAPI = `${apiServer}/api/flightctl`;
+const metricsAPI = `${apiServer}/api/metrics`;
+
 export const fetchMetrics = async (metricQuery: string, token: string | undefined, abortSignal?: AbortSignal) => {
   try {
-    const response = await fetch(`${apiServer}/metrics-range?query=${metricQuery}`, {
+    const response = await fetch(`${metricsAPI}/api/v1/query_range?${metricQuery}`, {
```

Comment:
There's a 400 error in this request, it should be 
```suggestion
    const response = await fetch(`${metricsAPI}/api/v1/query_range?query=${metricQuery}`, {
```

However, the UI cannot read the response from this request. The reason being that the server is returning multiple `Access-Control-Allow-Origin` headers. 

I'm guessing this may be coming from `corsHandler` but I don't know much Go to fix it properly.


![cors-headers](https://github.com/flightctl/flightctl-ui/assets/829045/0eb9eacf-5161-4ca0-a7b2-9bde4c4c589c)

Reply:
Great catch. We need to filter out `Access-Control` headers. Should be fixed now.